Makefile

```diff
@@ -73,6 +73,30 @@ downstream_local:
 	    rm -fr ${DOWNSTREAM_NAME}-${VERSION}.tar.gz
 	git checkout ${UPSTREAM_NAME}.spec
 
+update_template:
+	# Clone from upstream and put required location.
+	# Nodejs
+	curl -sL https://github.com/openshift/nodejs-ex/raw/master/openshift/templates/nodejs-mongodb.json > services/openshift/templates/common/nodejs-mongodb.json
+	curl -sL https://github.com/openshift/nodejs-ex/raw/master/openshift/templates/nodejs.json > services/openshift/templates/common/nodejs.json
+	# Cakephp
+	curl -sL https://github.com/openshift/cakephp-ex/raw/master/openshift/templates/cakephp.json > services/openshift/templates/common/cakephp.json
+	curl -sL https://github.com/openshift/cakephp-ex/raw/master/openshift/templates/cakephp-mysql.json > services/openshift/templates/common/cakephp-mysql.json
+	# Jenkins
+	curl -sL https://github.com/openshift/origin/raw/master/examples/jenkins/jenkins-ephemeral-template.json > services/openshift/templates/cdk/jenkins-ephemeral-template.json
+	curl -sL https://github.com/openshift/origin/blob/master/examples/jenkins/pipeline/jenkinstemplate.json > services/openshift/templates/adb/jenkinstemplate.json
+	curl -sL https://github.com/openshift/origin/raw/master/examples/jenkins/jenkins-persistent-template.json > services/openshift/templates/cdk/jenkins-persistent-template.json
```

Comment:
A small suggestion, can you please re-arrange the curl to down CDK related download near to each other?